geomesa-raster/src/test/scala/org/locationtech/geomesa/raster/data/RasterStoreQueryIntegratedTest.scala

```diff
@@ -129,7 +129,7 @@ class RasterStoreQueryIntegratedTest extends Specification {
 
       rasterStore must beAnInstanceOf[RasterStore]
       val theResults = rasterStore.getRasters(query).toList
-      theResults.length must beEqualTo(1)
+      theResults.length must beEqualTo(2)
```

Comment:
Why did the answer change here? Is the test pulling from a Store in one of the other tests?